src/scripts/OSFramework/OSUI/Pattern/Rating/Rating.ts

```diff
@@ -169,16 +176,20 @@ namespace OSFramework.OSUI.Patterns.Rating {
 		 * @memberof Rating
 		 */
 		private _renderItem(index: number): void {
-			// If first input, whihc is hidden, than also hide the label
-			const hideLabelClass: string = index === 0 ? Enum.CssClass.WCAGHideText : '';
 			// if not first input, which is hidden, add the html stored form the placeholders
 			const labelHTML = index !== 0 ? this._clonedPlaceholders : '';
 			// Create a unique rating input id, based on the index
 			const ratingInputId: string = this.uniqueId + '-rating-' + index;
 
-			// Craete input and label html
-			const input = `<input type="radio" class="${Enum.CssClass.RatingInput} ${Enum.CssClass.WCAGHideText}" id=${ratingInputId} name=${this._ratingInputName} value=${index}/>`;
-			const label = `<label class='${Enum.CssClass.RatingItem} ${hideLabelClass}' for=${ratingInputId}><span class="${Enum.CssClass.WCAGHideText}">Rating ${index}</span>${labelHTML}</label>`;
+			// Create input and label html
+			const input = `<input type="radio" class="${Enum.CssClass.RatingInput} ${Enum.CssClass.WCAGHideText}" id=${ratingInputId} name=${this._ratingInputName} value=${index} aria-hidden="true">`;
```

Comment:
It's possible to enum the types and aria roles? If it's possible, lets avoid this on our code and change to enums

Reply:
done